src/assets/i18n/fr.json

```diff
@@ -377,9 +377,9 @@
   "retreat-list-item.messageAlertWaitingList": "Vous devez vous connecter pour être sur notre liste d'attente",
   "retreat-list-item.notifications.subscribe_waiting_list.error.content": "Inscription échoué",
   "retreat-list-item.notifications.subscribe_waiting_list.error.title": "Erreur",
-  "retreat-list-item.notifications.subscribe_waiting_list.success.content_1": "Vous êtes maintenant inscrit à la liste d'attente.",
-  "retreat-list-item.notifications.subscribe_waiting_list.success.content_2": "Vous êtes en position",
-  "retreat-list-item.notifications.subscribe_waiting_list.success.title": "Succès!",
+  "retreat-list-item.notifications.subscribe_waiting_list.success.content_1": "Vous êtes en ",
+  "retreat-list-item.notifications.subscribe_waiting_list.success.content_2": "e position sur la liste d'attente.",
```

Comment:
Il y a une fonctionnalité sympa qui je pense serais bien ici: https://github.com/ngx-translate/core#3-define-the-translations

Avec l'usage de `{{ ma_variable }}` tu peux insérer le chiffre et donc traduire en une seule ligne de traduction.

Avantage : Lorsque tu va traduire en Anglais ou dans d'autre langue tu souhaite souvent deplacer la position du chiffre dans la phrase et c'est donc plus propre a maintenir avec les variables.

---

Aussi, le `1e`, `2e`, .., `14e` c'est un bon hack! Mais en Anglais ça ne va pas vraiment marcher. En réalité il nous faudrais un plugin de Pluralization comme [celui-ci ](https://github.com/lephyrus/ngx-translate-messageformat-compiler).

La pluralization est souvent complexe, donc la grande majorité du temps on tourne les phrases autrement pour l'éviter. Ici tu pourrais mettre `Vous êtes à la position {{ index }} de la liste d'attente.` par exemple.

Listes des plugins du package d'I18N: https://github.com/ngx-translate/core#plugins